PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeRequest.java

```diff
@@ -203,6 +203,21 @@ public void setReturnUrl(@NonNull String returnUrl) {
         this.returnUrl = returnUrl;
     }
 
+    @Nullable
+    public String getUserAuthenticationEmail() {
+        return authEmail;
+    }
+
+    /**
+     * Optional: Sets the email of the user prior to launching authentication
```

Comment:
Can we use the same docstring across [iOS](https://github.com/braintree/braintree_ios/pull/1118) and Android?